docs/reference/index.rst

```diff
@@ -0,0 +1,10 @@
+*******************
+xrtpy API Reference
```

Comment:
This goes for all of the pages. I don't think it is necessary to have the package name in each of the page titles. This helps to decluttter the title bar as well.

```suggestion
API Reference
```

Reply:
Changed.